server/routes/stats_router.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ResponseError } from '@opensearch-project/opensearch/lib/errors';
+import {
+  IOpenSearchDashboardsResponse,
+  IRouter,
+  SavedObjectsFindResponse,
+} from '../../../../src/core/server';
+import { APP_API, APP_PATH, MAP_SAVED_OBJECT_TYPE } from '../../common';
+import { getStats } from '../common/stats/stats_helper';
+import { MapSavedObjectAttributes } from '../../common/map_saved_object_attributes';
+
+export function statsRoute(router: IRouter) {
+  router.get(
+    {
+      path: `${APP_API}${APP_PATH.STATS}`,
+      validate: false,
+    },
+    async (
+      context,
+      request,
+      response
+    ): Promise<IOpenSearchDashboardsResponse<any | ResponseError>> => {
+      try {
+        const savedObjectsClient = context.core.savedObjects.client;
+        const mapsSavedObjects: SavedObjectsFindResponse<MapSavedObjectAttributes> =
+          await savedObjectsClient?.find({ type: MAP_SAVED_OBJECT_TYPE });
+        const stats = getStats(mapsSavedObjects);
+        return response.ok({
+          body: stats,
+        });
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.error(error);
```

Comment:
I understand this helps understand the error, but in production we don't log things. This can be potential for code execution. Can we check with dashboard team on how to log error messages?

Reply:
We can see there are some other places already use  in OSD or plugins. https://github.com/search?q=org%3Aopensearch-project+console.error%28error%29%3B&type=code

Reply:
we have some plugins not following patterns. the standard way is to use logger object from osd: https://github.com/opensearch-project/dashboards-reporting/blob/df8b413ae12ee6ac237392246a9da4e5e858eac5/server/routes/report.ts#L63

Reply:
Thanks, will update